styles/_base.scss

```diff
@@ -170,6 +178,16 @@ ul {
   @include max-width-container;
 }
 
+.fit {
+  max-width: 500px;
```

Comment:
I think we should resize the image and drop this class. I should probably do that on the live Artstor site as well.

Reply:
@anjanette yeah that makes a lot of sense.